File: Unidad3/C#/Programa16.cs
using System;

class Program
{
    static void Main()
    {
        for (int employee = 1; employee <= 20; employee++)
        {
            Console.Write("Ingrese las horas trabajadas para el empleado #" + employee + ": ");
            int horasTrabajadas = int.Parse(Console.ReadLine);

            Console.Write("Ingrese el pago por hora para el empleado #" + employee + ": ");
            double pagoPorHora = double.Parse(Console.ReadLine);

            double sueldoSemanal = horasTrabajadas * pagoPorHora;
            Console.WriteLine("El sueldo semanal del empleado #" + employee + " es: $" + sueldoSemanal);
        }
    }
}

File: Unidad3/Java/Programa16.java
package com.mycompany.programa16;
import java.util.Scanner;
public class Programa16 {

    public static void main(String[] args) {
        Scanner sc = new Scanner (System.in);
        int c=1, horas, pago, sueldo;
        while (c<=20){
        System.out.println("captura las horas trabajadas:");
        horas=sc.nextInt();
        System.out.println("captura el pago por hora:");
        pago=sc.nextInt();
        sueldo=horas*pago;
        System.out.println("Su sueldo es:"+sueldo);
        c++;
                }
    }
}
